int32             object_count
RadarTarget[100]  radar_objects